docs/1.0. Certificate Provisioning.md

```diff
@@ -206,6 +206,17 @@ An EST Client MUST provide a method to replace the manufacturer-issued TLS Clien
 
 An EST Client MAY ignore all timestamps in the certificate validity periods during the certificate provisioning process if it does not accurately know the current time, see [Bootstrapping Remote Secure Key Infrastructures - Section 2.6.1](https://tools.ietf.org/html/draft-ietf-anima-bootstrapping-keyinfra-41#section-2.6.1) for more information.
 
+If an EST Client implements the NMOS IS-04 Node API, then the EST Client SHOULD advertise that it supports BCP-003-03 using the 'caps' attribute of the Node API. Control interfaces can identify all Nodes which support BCP-003-03 by a key 'cert' and Uniform Resource Name (URN) having the base 'urn:x-nmos:control:cert' followed by a '/' character and the supported version.
```

Comment:
```suggestion
If an EST Client implements the NMOS IS-04 Node API, then the EST Client SHOULD advertise that it supports BCP-003-03 using the 'caps' attribute of the Node API. Control interfaces can identify all Nodes which support BCP-003-03 by a key 'cert' and Uniform Resource Name (URN) having the base 'urn:x-nmos:control:cert' followed by a '/' character and the supported version of this BCP.
```

We probably need a mechanism to cope with a client which supports multiple versions of the BCP. For Device 'controls' the array allows us to have multiple entries with different versioning.